### PR TITLE
Add writing section

### PR DESCRIPTION
**Pull request by kandebonfim**
##### Demo

captura de tela 2016-10-27 as 17 33 58

Please review @startae/frontend-team

### CODE REVIEW

**Comment by renatocarvalho** [comment]
- I think we can Merge the Guidelines and the How to.
- The questions could be the last item on the menu

**Comment by kandebonfim** [comment]
@renatocarvalho done!

File: source/writing/guidelines.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Use natural, conversational language, no bullshit.
+- Contractions are okay to be used. "Here's a list of projects we’ve released & contributed to." Instead of "Here is a list of projects we have released & contributed to". They make the text more human
+- Avoid developer/designer-focused terminology, unless you’re writing for them.
+
+#### Be direct
+- Use active voice "The boy hit the ball" not "The ball was hit by the boy".
+- Avoid complex verb structures.
+
+#### Be Positive
+-  Remember, we empower positive change.
```
Comment (sales) [inline source/writing/guidelines.md:25]: There's an extra space here

File: source/writing/guidelines.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Always try to write sentences positively, not negatively.
+
+# Basic how-to
+
+If you don't read anything else just read this:
+
+- When you are writing here you are representing Startaê. You are speaking for yourself and for it
+- Think problem are you solving to who is reading and how are you solving it
```
Comment (sales) [inline source/writing/guidelines.md:33]: This sentence is unclear to me. I'm not sure if I got, but how about something like:

> Think about the problem you are solving in a perspective of the reader, and how are you solving it

File: Gemfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```
Comment (sales) [inline Gemfile:1]: I think we can keep the `https` versioned. If someone is having problems with the `ssl` locally, just change to install/update the gems, but doesn't commit.

File: Gemfile.lock [hunk 4]
```diff
@@ -1,5 +1,5 @@
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.org/
```
Comment (sales) [inline Gemfile.lock:2]: same here

File: Gemfile (hunk 3 above)
Comment (renatocarvalho) [reply Gemfile:1]: True

**Pull request merged**